releases/sui/2025-01-ITS-v1.1.0.md

```diff
@@ -80,55 +79,43 @@ kubectl exec genesis-0 -- axelard tx wasm execute "$ITS_HUB_ADDR" \
 --gas-adjustment 1.2
 ```
 
-#### Stagenet/Testnet:
+#### Stagenet/Testnet/Mainnet:
+
+ITS hub contract configuration in json file must include the following attributes per chain:
 ```bash
-node cosmwasm/submit-proposal.js its-hub-register-chains sui -t "Register ITS for Sui chain" -d "Register ITS for Sui chain at ITS Hub contract" --deposit <100000000 for stagenet and 2000000000 for testnet> \
--m '{
-  "register_chains": {
-    "chains": [
-      {
-        "chain": "sui",
-        "its_edge_contract": "'"$ITS_SUI_CHANNEL"'",
-        "truncation": {
-          "max_decimals_when_truncating": '"8"',
-          "max_uint": "'"18446744073709551615"'"
-        }
+"axelar": {
+  "contracts": {
+    ...
+    "InterchainTokenService": {
+      ...
+      "some-sui-chain": {
+        "maxUintBits": 64,
+        "maxDecimalsWhenTruncating": 6
       }
-    ]
+    }
+    ...
   }
-}'
+}
 ```
 
-#### Mainnet:
 ```bash
-node cosmwasm/submit-proposal.js its-hub-register-chains sui -t "Register ITS for Sui chain" -d <Description along with community proposal for ITS integration> --deposit 2000000000 \
--m '{
-  "register_chains": {
-    "chains": [
-      {
-        "chain": "sui",
-        "its_edge_contract": "'"$ITS_SUI_CHANNEL"'",
-        "truncation": {
-          "max_decimals_when_truncating": '"8"',
-          "max_uint": "'"18446744073709551615"'"
-        }
-      }
-    ]
-  }
-}'
+node cosmwasm/submit-proposal.js its-hub-register-chains sui -t "Proposal title" -d "Proposal description" --deposit <100000000 for stagenet and 2000000000 for testnet/mainnet>
 ```
 
-### Setting up trusted addresses
+
+### Setting up trusted addresses on Sui
 ```bash
-ITS_HUB_ADDR=$(cat ./axelar-chains-config/info/<your_config_name>.json | jq -rM '.axelar.contracts.InterchainTokenService.address')
+node sui/its.js add-trusted-chains axelar
 
-node sui/its.js add-trusted-chains axelar $ITS_HUB_ADDR
+# Add all trusted chains to ITS Sui
+node sui/its.js add-trusted-chains all
+```
 
+### Setting up trusted addresses on EVM
+```bash
+# For these commands replace Sui's PRIVATE KEY in .env with private key for EVM chains
 # comma separated list of existing chains of ITS
-node evm/its.js -n avalanche-fuji,ethereum-sepolia --action setTrustedAddress --trustedChain sui --trustedAddress hub
-
-# the other way around, done for all existing chains with ITS
-node sui/its.js add-trusted-chains all-evm
+node evm/its.js -n all --action setTrustedAddress --trustedChain sui --trustedAddress hub
```

Comment:
Create a new command `node evm/its.js -n all set-trusted-chain sui` that sets trusted address to `hub` under the hood